examples/js/loaders/GLTF2Loader.js

```diff
@@ -1668,10 +1693,12 @@ THREE.GLTF2Loader = ( function () {
 	GLTFParser.prototype.loadMeshes = function () {
 
 		var json = this.json;
+		var extensions = this.extensions;
 
 		return this._withDependencies( [
 
 			"accessors",
+			"bufferViews",
```

Comment:
Correct – `bufferViews` are a dependency of `accessors`, so they're already loaded, but must also be listed explicitly here for them to be included in the `dependencies` parameter.